frontend/src/concepts/pipelines/content/configurePipelinesServer/PipelineDropdown.tsx

```diff
@@ -83,20 +85,19 @@ export const PipelineDropdown = ({
           onClick={onToggle}
           isExpanded={isOpen}
         >
-          <KeyIcon />
+          <Flex spaceItems={{ default: 'spaceItemsSm' }}>
+            <FlexItem>
+              <OptimizeIcon />
+            </FlexItem>
+            <FlexItem>Autofill from connection</FlexItem>
+          </Flex>
```

Comment:
This layout is not needed. You can simply pass the icon into `icon` prop of the `<MenuToggle>` component.